src/main/webapp/view.js

```diff
@@ -19,12 +19,10 @@ window.LECTURE_ID = getLectureId();
 
 initialize();
 
-/**
- * Initializes the discussion section of the lecture view page.
- */
+/** Initializes different portions of project */
```

Comment:
This comment is too vague. Does not really inform me on anything.

Reply:
removed